do-test.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
@rjrelyea You can ignore these scripts. They won't make it into the final NSS check in for obvious reasons.